.github/workflows/push_images.yml

```diff
@@ -0,0 +1,112 @@
+# This workflow collects screenshots which have stored as artifacts in test.yml 
+# and opens a PR on conda-store to update the docs.
+#
+# There are two ways to trigger this workflow:
+# 1. Trigger by adding the `update_screenshots` label. 
+#    The workflow will run immediately after you add the label. It is
+#    also expecting artifacts which have been generated in test.yml. 
+#    Therefore, this cannot be triggered until test.yml is complete. 
```

Comment:
Does this mean the reviewer or committer needs to wait until `tests.yml` is done then manually add the label?

Reply:
correct.